Makefile

```diff
@@ -27,4 +27,20 @@ all-tests:
 generate build clean:
 	go $@ $(EXTRAGOARGS)
 
+sandbox-test-fc-build:
```

Comment:
Assuming the image build is something a person would ever trigger manually, I'd suggest doing the actual work in a `sandbox-test-fc-build-stamp` rule. The final step of that rule's recipe should be to touch a file of the same name. Then add a `sandbox-test-fc-build` PHONY rule that doesn't do anything but declares a dependency on the `-stamp` variant. Having `stamp` in the file name makes the file's purpose a little more apparent.